tests/Feature/Controllers/CommentControllerTest.php

```
@@ -0,0 +1,113 @@
+        /* Arrange */
+        $task = Task::factory()->create(['user_assigned_id' => $this->user->id]);
+
+        /* Act */
+        $response = $this->post(route('comments.create', ['type' => 'task', 'external_id' => $task->external_id]), [
+            'description' => 'Test comment on task',
+        ]);
+
+        /* Assert */
+        $response->assertRedirect();
+        $this->assertDatabaseHas('comments', ['user_id' => $this->user->id]);
+    }
```

PHPMD (2.15.0)

[error] The method it_stores_comment_on_task is not named in camelCase. (undefined)

(CamelCaseMethodName)